xbmc/interfaces/legacy/RenderCapture.h

```diff
@@ -35,32 +34,36 @@ namespace XBMCAddon
 
     class RenderCapture : public AddonClass
     {
-      CRenderCapture* m_capture;
+      unsigned int m_captureId;
```

Comment:
sorry - had the pull from yesterday where this was not present ;) - i will push the changes to the addon in a couple.